lib/src/signed_video_tlv.c

```diff
@@ -902,6 +904,56 @@ tlv_find_tag(signed_video_t *self,
   return NULL;
 }
 
+svi_rc
+tlv_find_tag_and_decode_recurrent_tags(signed_video_t *self,
+    const uint8_t *tlv_data,
+    size_t tlv_data_size,
+    bool with_ep)
```

Comment:
emulation prevention bytes have already been removed if we use this on the auth side